angular.json

```diff
@@ -24,8 +24,10 @@
             ],
             "styles": [
               "node_modules/bootstrap/dist/css/bootstrap.min.css",
-              "node_modules/font-awesome/css/font-awesome.min.css",
-              "src/styles.css"
+              "src/styles.css",
+              "https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css",
+              "https://use.fontawesome.com/releases/v5.5.0/css/all.css" 
```

Comment:
There is no comma at the end of use.fontawesome.com link. Also, please test the changes done in angular.json locally.

Reply:
There need not to be a comma as it's the last file we're importing.

Reply:
Done